alchemist-web-renderer/build.gradle.kts

```
@@ -90,35 +89,24 @@ application {
 }
 
 tasks.getByName<JavaExec>("run") {
```

Does a plain:

```suggestion
tasks.run {
```

work?

In any case, avoid `getByName`, use the [configuration avoidance API](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html). I usually do not enforce it, but the build of Alchemist is already too slow to leave things sub-optimal.

---

Ok, I'll remove this